main.go

```diff
@@ -72,22 +73,16 @@ func createInstallNpmCommand() (*command.Model, error) {
 		return nil, fmt.Errorf("unsupported operating system: %s", runtime.GOOS)
 	}
 
-	return command.NewWithStandardOuts(args[0], args[1:]...), nil
-}
-
-func runAndLog(cmd *command.Model) error {
-	log.Donef(fmt.Sprintf("$ %s", cmd.PrintableCommandArgs()))
-	if err := cmd.Run(); err != nil {
-		return fmt.Errorf("error running npm command: %s", err)
-	}
-
-	return nil
+	return command.New(args[0], args[1:]...), nil
 }
 
 func setNpmVersion(ver string) error {
-	cmd := command.NewWithStandardOuts("npm", "install", "-g", fmt.Sprintf("npm@%s", ver))
-	err := runAndLog(cmd)
-	if err != nil {
+	cmd := command.New("npm", "install", "-g", fmt.Sprintf("npm@%s", ver))
```

Comment:
NOTE: `fmt.Sprintf("npm@%s", ver)` could be shorter by: `"npm@"+ver`

Reply:
I agree. I'll use concatenation for simple string combining scenarios in the future, but I prefer to have this PR merged for now, if you don't mind.